Docker/README.md

```diff
@@ -75,16 +75,23 @@ To run analysis __in CI__:
    The output of `output-directory` is described [here](../General/output.md#basic-output).  
    Consider using [fail-thresold](../General/qodana-yaml.md#fail-threshold) to make build fail on certain number of problems reached. [Running as non-root](techs.md#run-as-non-root) is also supported.
    
- - Example for GitHub Action (`.github/workflows/qodana.yml`):
+ - Example for GitHub Workflow (`.github/workflows/qodana.yml`):
    ```yaml
    jobs:
      qodana:
        runs-on: ubuntu-latest
        steps:
          - uses: actions/checkout@v2
+         - uses: actions/cache@v2
+           with:
+             path: ~/work/_temp/_github_home/cache
+             key: ${{ runner.os }}-qodana-${{ github.ref }}
+             restore-keys: |
+               ${{ runner.os }}-qodana-${{ github.ref }}
+               ${{ runner.os }}-qodana-   
          - uses: docker://jetbrains/qodana
            with:
-             args: --results-dir=/github/workspace/qodana --save-report --report-dir=/github/workspace/qodana/report
+             args: --cache-dir=/github/home/cache --results-dir=/github/workspace/qodana --save-report --report-dir=/github/workspace/qodana/report
```

Comment:
Why did you decide to use `/github/home/cache` as a `cache-dir`?

Reply:
We'd better use some other dir for `cache` instead of `project-dir`, because in other case we would have to exclude cache subdir of project from inspection checks.
And this `/github/home` is the only other directory apart from project dir which is mapped to container.